internal/storage/v2/elasticsearch/tracestore/fixtures/otel_traces_01.json

```diff
@@ -0,0 +1,127 @@
+{
+  "resourceSpans": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "service.name",
+            "value": {
+              "stringValue": "service-x"
+            }
+          },
+          {
+            "key": "peer.ipv4",
+            "value": {
+              "intValue": "23456"
+            }
+          },
+          {
+            "key": "error",
+            "value": {
+              "boolValue": true
+            }
+          }
+        ]
+      },
+      "scopeSpans": [
+        {
+          "scope": {},
```

Comment:
should have something in there, like scope name/version at least, but also some attributes to test conversion